pkg/datarepair/checker/checker.go

```diff
@@ -14,23 +15,27 @@ import (
 	"storj.io/storj/pkg/node"
 	"storj.io/storj/pkg/pb"
 	"storj.io/storj/pkg/pointerdb"
+	"storj.io/storj/pkg/utils"
 	"storj.io/storj/storage"
 )
 
 // Checker is the interface for the data repair queue
 type Checker interface {
 	IdentifyInjuredSegments(ctx context.Context) (err error)
 	Run() error
-	Stop() error
 }
 
 // Checker contains the information needed to do checks for missing pieces
 type checker struct {
+	ctx         context.Context
```

Comment:
I don't think it's typically a good idea to put the context on the struct. We should pass it between functions.

Reply:
Yeah. We are changing the repairer to just pass the ctx to the repairier.Run() method. Checker should also do that